src/lib/ruby-to-blocks-converter/microbit.js

```diff
@@ -1,10 +1,151 @@
+                        block, 'GESTURE', 'microbit_menu_gestures', 'gestures',
+                        args[1], 'moved'
+                    )
+                    this._setParent(rubyBlock, block);
+                }
+                break;
+            case 'microbit_tilted':
+                if (args.length === 2 && this._isString(args[1])) {
+                    block = this._createBlock('microbit_whenTilted', 'hat');
+                    this._addFieldInput(
+                        block, 'DIRECTION', 'microbit_menu_tiltDirectionAny', 'tiltDirectionAny',
+                        args[1], 'any'
+                    )
+                    this._setParent(rubyBlock, block);
+                }
+                break;
+            case 'microbit_pin_connected':
+                if (args.length === 2 && this._isNumber(args[1])) {
+                    block = this._createBlock('microbit_whenPinConnected', 'hat');
+                    this._addFieldInput(
+                        block, 'PIN', 'microbit_menu_touchPins', 'touchPins',
+                        args[1], '0'
+                    )
+                    this._setParent(rubyBlock, block);
+                }
+                break;
+            }
+        } else if (this._isSelf(receiver) || receiver === Opal.nil) {
+            switch (name) {
+            case 'microbit':
+                if (args.length === 0) {
+                    block = this._createRubyExpressionBlock(MicroBit);
+                }
+                break;
+            }
+        } else if (this._equalRubyExpression(receiver, MicroBit)) {
+            switch (name) {
+            case 'button_pressed?':
+                if (args.length === 1 && this._isString(args[0])) {
+                    block = this._changeBlock(receiver, 'microbit_isButtonPressed', 'value_boolean');
+                    delete this._context.blocks[receiver.inputs.EXPRESSION.block];
+                    delete receiver.inputs.EXPRESSION;
+
+                    this._addFieldInput(
+                        block, 'BTN', 'microbit_menu_buttons', 'buttons',
+                        args[0], 'A'
+                    );
+                }
+                break;
+            case 'display':
+                make_display_block:
+                if (args.length === 5) {
+                    for (const arg of args) {
+                        if (!this._isString(arg)) {
+                            break make_display_block;
+                        }
+                    } 
+
+                    block = this._changeBlock(receiver, 'microbit_displaySymbol', 'statement');
+                    delete this._context.blocks[receiver.inputs.EXPRESSION.block];
+                    delete receiver.inputs.EXPRESSION;
+
+                    var matrix = '';
+                    for (const arg of args) {
+                        matrix += arg; 
+                    } 
+                    matrix = matrix.replace(/\./g, '0');
```

Comment:
仕様は、1〜9はLEDを点灯、それ以外はLEDを消灯です。
そのため、以下のようにすべきです。

```suggestion
                    matrix = matrix.replace(/[1-9]/g, '1').replace(/[^1-9]/g, '0');
```